src/main/java/vfyjxf/bettercrashes/utils/GuiProblemScreen.java

```diff
@@ -105,16 +107,15 @@ protected void actionPerformed(GuiButton button) {
             }
         }
         if (button.id == 2) {
-            if (hasteLink == null) {
+            if (pasteLink == null) {
                 button.enabled = false;
                 button.displayString = I18n.format("bettercrashes.gui.common.uploading");
                 Thread thread = new Thread("BetterCrashes report uploading") {
 
                     @Override
                     public void run() {
                         try {
-                            hasteLink = CrashReportUpload
-                                    .uploadToUbuntuPastebin("https://paste.ubuntu.com", report.getCompleteReport());
+                            pasteLink = CrashReportUpload.uploadToPastebin(report.getCompleteReport());
```

Comment:
It's my fault but can you add `setClipboardString(pasteLink.toString());` here to restore feature to set url to clipboard when upload button is clicked, not only when clicking "success" button?